sklearn/semi_supervised/label_propagation.py

```diff
@@ -342,6 +343,22 @@ class LabelPropagation(BaseLabelPropagation):
     LabelSpreading : Alternate label propagation strategy more robust to noise
     """
 
+    def __init__(self, kernel='rbf', gamma=20, n_neighbors=7,
+                 alpha=None, max_iter=30, tol=1e-3, n_jobs=1):
+        # alpha is deprecated for LabelPropagation because it doesn't have any
+        # theoretical meaning (from the reference paper). Look at PR 6727.
+        if alpha is not None:
+            warnings.warn(
+                "Deprecated in 0.19 and it's going to be removed in 0.21.",
+                DeprecationWarning
+            )
+        alpha = .0
```

Comment:
Ok, I'll set the `alpha` of the line below to `.0`.

But what about the `alpha` of the constructor above? You said it should be set to `None`, but the tests fail because the two constructors have different default values (`None` and `0`).

What am I missing?